apis/certificateregistry/certificateregistryapi.yaml

```diff
@@ -0,0 +1,376 @@
+swagger: '2.0'
+info:
+  version: '1.0'
+  title: Certificate api
+  description: >-
+    This api is used for store download validate the course certificate.
```

Comment:
Certificate registry is used for storing , downloading and validation of certificate.

Reply:
done